### DISCUSSION TITLE
Adding OCR with teseract

### DISCUSSION THREAD

**Discussion by Win6old** [nextcloud/all-in-one#751] [Ideas] [answered]
It would be greate when you could enable one of the Nextcloud OCR Apps in the container stack

**Comment by szaimen** [comment]
Which app? Can you post the appstore link?

**Reply by giovacode** [reply]
https://apps.nextcloud.com/apps/workflow_ocr

**Comment by szaimen** [comment]
This will not be integrated into AIO.

**Reply by GuillaumeHullin** [reply]
Hello @szaimen, 
Is there a technical reason, lack of time or some other explanation why it would not be integrated to AIO?
I'm very green to NextCloud (deployed it a few days ago) and I actually was looking for a way to get OCR working with my test install of AIO...

**Reply by szaimen** [reply]
Mostly this is the reason: https://github.com/nextcloud/all-in-one/discussions/1159#discussioncomment-3655824

**Reply by giovacode** [reply]
@szaimen i understand your fears and appreciate this safe approach.
As OCR would be a **huge** value for nextcloud (as FullText Search), i am doing a try.

I am not an docker expert, so i have just added dependancies to Dockerfile


then overriten nextcloud-aio-nextcloud :latest image
![image](https://user-images.githubusercontent.com/73257680/190514557-397ece70-5641-4f25-861f-26bdd2b55877.png)

I'm sure this is not the thing to do, this is just to make a test.

**Comment by szaimen** [comment, marked as answer]
This will be handled with https://github.com/nextcloud/all-in-one/issues/1162

**Reply by szaimen** [reply]
This is now released with v3.0.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
